packages/vm/lib/state/interface.ts

```diff
@@ -10,7 +10,8 @@ export interface StorageDump {
 export interface StateManager {
   copy(): StateManager
   getAccount(address: Buffer): Promise<Account>
-  putAccount(address: Buffer, account: Account): Promise<void>
+  putAccount(address: Buffer, account: Account | null): Promise<void>
```

Comment:
I don't see the interface of `DefaultStateManager.putAccount` changing. Is this change still valid or maybe `deleteAccount` suffices here?

Reply:
Yup I should revert the changes to the interface, thanks!